email2phonenumber.py

```diff
@@ -19,654 +24,909 @@
+					"BNEEdl5uJFiKHQSy%2F5WAFuuX3yerQHq%2BYsDdd0f4lWftNcHxdLJ5C1IKhm7"
+					"TLEAbt37Szs%2FNf1lxuQJ4H8GWolMSKCXjPTr7JX5lmCbLaM6cqPkYsoMfkYZK"
+					"HcTB5qL3Fuc%2F1RjRCVbebr5eqK8mEHq6miZH%2FkNYCwMhG%2Fe8dk9pD8wcQ"
+					"m9dAoX4LyiGmcFSHkw7Vyu3c%2FyEFRKQr5McsOyPavoTIl73JelEUq2gjfXw%2"
+					"Fj6BMGNWhttumOOPOkNcRAUUZoNvKo44x%2BNGUVNN0hgYmrDt9FBx%2F%2F2Pf"
+					"HDNu3TsOZCqB5LyPgmCiZ83StGq2dy%2FX8hlWXdBfcvJdNTCRrob7GIYrZg3so"
+					"md0r%2FbS2tYsIPj6OLAhZf9EuZ8sa6ECrFp9Lhl321R8ls3kS3zyIFNMEWWFFW"
+					"OMxRrTiZ%2FYUCi4%2FkFrEuPR%2BTpWb%2Bv%2FjCVyA6xXvdrHThl%2FBPMHI"
+					"L%2FNsH3%2Fc8xEHoaHiEpuMnmYRdr7Kg8vC%2FrE0Q5gGxHa%2BzgfX1w0Snzy"
+					"mr%2BNZyq8lKNwidFciPR30pkKoS4VsZpzq7WlxT1k8rSH25Iqi20%2Bg%3D%3D",
+					allow_redirects=False,
+					proxies=proxy,
+					verify=verifyProxy
+				)
+				redirect_prev_rid = re.search('prevRID=([A-Z0-9]+)&', redirect_url).group(1)
+				redirect_param_jwt = re.search('paramJwt=(.*)&', redirect_url).group(1)
+				response = session.get(
+					redirect_url,
+					headers={
+						"Cache-Control": "max-age=0",
+						"Upgrade-Insecure-Requests": "1",
+						"User-Agent": user_agent,
+						"Accept":
+							"text/html,application/xhtml+xml,application/xml;q=0.9,"
+							"image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3",
+						"Referer": "https://www.amazon.com/ap/forgotpassword?openid.assoc_handle=usflex",
+						"Accept-Language": "en-US,en;q=0.9"
 					},
-					proxies = proxy,
-					verify = verifyProxy)
+					proxies=proxy,
+					verify=verifyProxy
+				)
 			else:
-				response = session.get("https://www.amazon.com" + redirectURL,
-					headers = {"Cache-Control": "max-age=0",
-								"Upgrade-Insecure-Requests": "1",
-								"User-Agent": userAgent,
-								"Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3",
-								"Referer": "https://www.amazon.com/ap/forgotpassword?openid.assoc_handle=usflex",
-								"Accept-Encoding": "gzip, deflate",
-								"Accept-Language": "en-US,en;q=0.9"
+				response = session.get(
+					"https://www.amazon.com" + redirect_url,
+					headers={
+						"Cache-Control": "max-age=0",
+						"Upgrade-Insecure-Requests": "1",
+						"User-Agent": user_agent,
+						"Accept":
+							"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,"
+							"image/apng,*/*;q=0.8,application/signed-exchange;v=b3",
+						"Referer": "https://www.amazon.com/ap/forgotpassword?openid.assoc_handle=usflex",
+						"Accept-Encoding": "gzip, deflate",
+						"Accept-Language": "en-US,en;q=0.9"
 					},
-					proxies = proxy,
-					verify = verifyProxy)
-
-
-		if response.status_code >= 500: # Error, let's try again
-			print YELLOW + "WARNING: 500 error returned for phone: " + phoneNumber + ENDC
+					proxies=proxy,
+					verify=verifyProxy
+				)
+		if response.status_code >= 500:
+			logger.warning("%s Error 500 returned for phone: %s %s", YELLOW, phone_number, ENDC)
```

Comment:
In many places where logging occurs, there is a space between %s and the first word. That results in odd spacing in temrinal output. It shoudl be "%sError 500..." instead of "%s Error 500..." to be consistent.

Reply:
Forgot to remove some of them, I think its now solved